x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/operator/topn/TopNOperator.java

```diff
@@ -548,9 +553,11 @@ public Page getOutput() {
         if (output == null || output.hasNext() == false) {
             return null;
         }
+        long start = System.nanoTime();
```

Comment:
I think we should capture the emit time in `finish()` (`toPages`) method instead?

Reply:
Sure.

Reply:
Looking more closely - of course. Wow. I wasn't paying attention. Thanks!